app/global.css

```
@@ -0,0 +1,35 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:ital,opsz,wght@0,14..32,100..900;1,14..32,100..900&display=swap');
+@import url('styles/_css-reset.css');
+@import url('styles/_semantic.css');
+@import url('styles/_utils.css');
+@import url('styles/_typography.css');
```

а в чем разница между 

```suggestion
@import 'https://fonts.googleapis.com/css2?family=Inter:ital,opsz,wght@0,14..32,100..900;1,14..32,100..900&display=swap';
@import 'styles/_css-reset.css';
@import 'styles/_semantic.css';
@import 'styles/_utils.css';
@import 'styles/_typography.css';
```

ну т.е. мож она и есть, я правда не в курсе